config/crd/bases/embeddedcluster.replicated.com_installations.yaml

```diff
@@ -89,6 +89,17 @@ spec:
                 properties:
                   extensions:
                     properties:
+                      builtin:
+                        description: |-
+                          BuiltinExtensions allows vendors to enable/disable additional optional extensions.
```

Comment:
This isn't true anymore.

Reply:
I'm going to say "it will be true again in the future" when we add distributed storage 😆

Reply:
Eventual consistency